motmetrics/distances.py

```diff
@@ -39,8 +37,8 @@ def norm2squared_matrix(objs, hyps, max_d2=float('inf')):
         Distance matrix containing pairwise distances or np.nan.
     """
 
-    objs = np.atleast_2d(objs).astype(float)
-    hyps = np.atleast_2d(hyps).astype(float)
```

Comment:
[comment] メッセージ的にはここは関係なさそうです。
厳密には`(Python)float`に対応するのは`np.float64`かと思います。
`/root/local/python-3.8.7/lib/python3.8/site-packages/motmetrics/distances.py:79: DeprecationWarning: np.float is a deprecated alias for the builtin float. To silence this warning, use float by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use np.float64 here.`